src/DrupalInfo.php

```diff
@@ -51,6 +51,14 @@ public function activate(Composer $composer, IOInterface $io)
     {
         $this->composer = $composer;
         $this->io = $io;
+
+        $config = $this->composer->getConfig();
+        if ($config) {
+            $additionalPackageTypes = $config->get('drupal-info-rewrite--additional-packageTypes');
```

Comment:
Does `drupal-info-rewrite--additional-packageTypes` get specified in `composer.json`? Seems like a very verbose string name :)

Reply:
Yes, that is where it goes. I don't care what the string is; it can be anything. I choose that so it would be specific to this plugin.

Reply:
Could you add a small sample to the `README.md` file so folks know this can be overridden? Once that is done, I think this is good to go!

Reply:
I just committed some documentation. Is this fine or do you want an actual example `composer.json` file?